packit_service/worker/handlers/distgit.py

```diff
@@ -167,6 +170,7 @@ class AbstractSyncReleaseHandler(
     helper_kls: type[SyncReleaseHelper]
     sync_release_job_type: SyncReleaseJobType
     job_name_for_reporting: str
+    get_dashboard_url: staticmethod
```

Comment:
Wouldn't it be better to type the attribute as `Callable[[int], str]` rather than `staticmethod`?